works.yaml

```diff
@@ -2248,18 +2238,6 @@ Jester:
   - MIT/X11 License
   license_reference:
   - https://github.com/dom96/jester/blob/master/license.txt
-JQuery:
-  uri:
-  - https://jquery.org/
-  tags:
-  - library
-  - software
-  - ui
-  - web
-  license:
-  - MIT/X11 License
-  license_reference:
-  - https://jquery.org/license/
```

Comment:
Please add this project back in, with this new license reference:

https://github.com/jquery/jquery/blob/master/LICENSE.txt